src/vorta/__main__.py

```diff
@@ -5,13 +5,24 @@
 import peewee
 from vorta._version import __version__
 from vorta.config import SETTINGS_DIR
-from vorta.log import init_logger
+from vorta.log import init_logger, logger
 from vorta.models import init_db
 from vorta.updater import get_updater
 from vorta.utils import parse_args
 
 
+def exception_handler(type, value, tb):
+    # https://stackoverflow.com/questions/49065371/why-does-sys-excepthook-behave-differently-when-wrapped
+    # This double prints the exception, want to only print the log entry
+    logger.critical("Uncaught exception, file a report at https://github.com/borgbase/vorta/issues/new:",
+                    exc_info=(type, value, tb))
+    sys.__excepthook__(type, value, tb)
+    sys.exit(1)
```

Comment:
Remove this line to make uncaught exceptions never crash, may lead to undesired behaviour.

Reply:
Fine. Lets crash it all the time. At least we get bug reports quickly.

Reply:
I've made the exception print a notification. Should I still create a crash?